qa/rpc-tests/wallet_protectcoinbase.py

```diff
@@ -114,11 +114,23 @@ def run_test (self):
         recipients = []
         recipients.append({"address":myzaddr, "amount": Decimal('20.0') - Decimal('0.0001')})
         myopid = self.nodes[0].z_sendmany(mytaddr, recipients)
-        self.wait_and_assert_operationid_status(myopid)
+        mytxid = self.wait_and_assert_operationid_status(myopid)
         self.sync_all()
         self.nodes[1].generate(1)
         self.sync_all()
 
+        # Verify that debug=zrpcunsafe logs params, and that full txid is associated with opid
+        logpath = self.options.tmpdir+"/node0/regtest/debug.log"
+        logcounter = 0
+        with open(logpath, "r") as myfile:
+            logdata = myfile.readlines()
+        for logline in logdata:
+            if myopid + ": z_sendmany initialized" in logline and mytaddr in logline and myzaddr in logline:
+                logcounter = logcounter + 1
+            if myopid + ": z_sendmany finished" in logline and mytxid in logline:
+                logcounter = logcounter + 1
+        assert_equal(logcounter, 2)
```

Comment:
This test doesn't check that the lines occur in the correct order. I think something like this would work:

```py
initcounter = 0
fincounter = 0
for logline in logdata:
    if myopid + ": z_sendmany initialized" in logline and mytaddr in logline and myzaddr in logline:
        initcounter = initcounter + 1
    if myopid + ": z_sendmany finished" in logline and mytxid in logline:
        assert_equal(initcounter, 1)
        fincounter = fincounter + 1
assert_equal(initcounter, 1)
assert_equal(fincounter, 1)
```